components/brave_federated/data_stores/data_store.cc

```diff
@@ -44,65 +44,152 @@ void DatabaseErrorCallback(sql::Database* db,
     DLOG(FATAL) << db->GetErrorMessage();
 }
 
+void BindCovariateToStatement(
+    const brave_federated::mojom::Covariate& covariate,
+    int training_instance_id,
+    base::Time created_at,
+    sql::Statement* s) {
+  s->BindInt64(0, training_instance_id);
+  s->BindInt64(1, static_cast<int>(covariate.type));
+  s->BindInt64(2, static_cast<int>(covariate.data_type));
+  s->BindString(3, covariate.value);
+  s->BindInt64(4, created_at.ToInternalValue());
```

Comment:
`ToInternalValue` should not be used, see the comment in base/time/time.h header

Reply:
See `FromDeltaSinceWindowsEpoch` and `ToDeltaSinceWindowsEpoch` for serialization/deserialization of base::Time